scripts/averages.js

```diff
@@ -0,0 +1,44 @@
+var pry = require('pryjs');
+
+function total(array) {
+  var sum = array.reduce(function (total, number) {
+    return total + number;
+  }, 0);
+return sum
+};
+
+function mean(array) {
+  var total = array.reduce(function (total, number) {
```

Comment:
hmmmm i imagine it may be confused with the fact that you've named a variable the same name as a function? could you calling total by passing the array in first then finding the mean? maybe something like this...

```js
function mean(array) {
  var totalArray = total(array)
  var length = array.length
  return totalArray / length
};

```